packages/airnode-node/src/evm/verification/api-call-verification.test.ts

```diff
@@ -26,7 +26,7 @@ describe('verifyApiCallIds', () => {
       chainId: '31337',
       requesterAddress: '0x7f7d1Aa0792aC39f43C6e7FA2ec31258Fc5FD612',
       sponsorAddress: '0x69e2B095fbAc6C3f9E528Ef21882b86BF1595181',
-      sponsorWalletAddress: '0xd5e6a768f1d23d30B386Bb5c125DBe83A9c40c73',
+      sponsorWalletAddress: '0xd5e6a768f1d23d30B386Bb5c125DBe83A9c40c73', // Invalid sponsor wallet address
```

Comment:
Does this just mean the test data is technically invalid? Is it possible to update these to use a valid address?

Reply:
Yeah, it would - but I think some test would break, since on places which need the correct sponsor wallet we use the overrides for it. 

I think it would be nicer if it was switched (and we passed) the invalid value when it's needed.

Reply:
Created https://api3dao.atlassian.net/browse/AN-447